Gem/Onboarding/Scenes/ImportWalletScene.swift

```diff
@@ -65,6 +66,19 @@ struct ImportWalletScene: View {
                                     }
                                 }
                                 .padding(.top, 8)
+                        case .privateKey:
+                            if let types = model.chain?.keyEncodingTypes, !types.isEmpty {
+                                let placeholder = types.map { $0.rawValue }.joined(separator: " / ")
+                                TextField(Localized.Wallet.Import.privateKey(placeholder), text: $privateKey, axis: .vertical)
+                                    .textInputAutocapitalization(.never)
+                                    .lineLimit(8)
+                                    .keyboardType(.alphabet)
```

Comment:
```suggestion
                                    .keyboardType(.alphabet)
```

default keyboard? because base64 also has number